### PR TITLE
Override API key

### PR DESCRIPTION
**Pull request by 99ff00**
Override hardcoded API key.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +90,12 @@ allprojects {
 
 ## API Reference
 
+#### setApiKey(): void
```
Comment (jukben) [inline README.md]: You have forgotten `setApiKey(apiKey: string): void`, haven't it?

Comment (99ff00) [reply README.md]: Indeed! Fixed.

**Comment by mike866** [comment]
Can you explain the usecase for having API key in JS? Do you think someone will need to use multiple API keys?

**Comment by 99ff00** [comment]
1. The app is getting configs from server
2. The app source code is used in third-party SDK and OT keys are different for every end-user
3. It's always nice to have flexible option like this

**Comment by jonathanwmaddison** [comment]
This PR will be useful. As @99ff00 mentioned, we are getting our configs from server, setting the api key will allow us more flexibility with changing our tok box project settings. Thanks for putting this together @99ff00!

**Pull request merged**